src/components/PostCard.tsx

```diff
@@ -0,0 +1,35 @@
+const PostCard = ({ post }) => {
+  return (
+    <article className="bg-arg-white p-6 sm:p-8 rounded border-4 border-arg-muted shadow-md shadow-arg-muted/40 grid gap-4">
+      <div>
+        <time
+          dateTime={new Date(post.data.date).toISOString()}
+          className="uppercase text-arg-gold"
+        >
+          {new Date(post.data.date).toLocaleDateString("en-us", {
+            year: "numeric",
+            month: "short",
+            day: "numeric",
+          })}
+        </time>
+        <h2 className="text-xl sm:text-2xl max-w-heading font-bold">
+          <a
+            className="font-bold focus:outline-none focus-visible:ring-4 ring-offset-2 ring-offset-arg-white ring-arg-gold"
+            href={`/${post.slug}/`}
```

Comment:
Do we want/need the trailing slash here? My guess is no? 
```suggestion
            href={`/${post.slug}`}
```

Reply:
I think it creates a redirect if you don’t do the trailing slash since Astro puts it in by default (if I'm remembering correctly); might be worth testing